lib/rmt/cli/repos.rb

```diff
@@ -28,7 +28,7 @@ def disable(id)
   protected
 
   def list_repositories(scope: :enabled)
-    repositories = (scope == :all) ? Repository.only_scc : Repository.only_scc.only_mirrored
+    repositories = ((scope == :all) ? Repository.only_scc : Repository.only_scc.only_mirrored).order(:name, :description)
```

Comment:
Why `:description`? Maybe there's something more suitable for sorting, arch or friendly name maybe?

Reply:
Well, it only makes sense to order by stuff that is also visible in the output, which would be:
https://github.com/SUSE/rmt/blob/3f45944df9ed56e37d052fda544e967860b563a7/lib/rmt/cli/repos.rb#L40-L47

Reply:
Well, description is basically `#{name} for #{arch}`, so sorting by description is equivalent to sorting by arch when names are the same.